### PR TITLE
feat(Card): add textAlign prop to Card subcomponents

### LINKED ISSUE #2027: Card subcomponents are missing `textAlign` props
Per SUI core, a Card's children can be aligned:

https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/components/card.css#L203

```jsx
<Card>
  <Card.Content textAlign="center">
    This should be centered.
  </Card.Content>
</Card>
```

This is true for all Card subcomponents.

### PR DESCRIPTION
**Pull request by itamar244**
Fixes #2027

### CODE REVIEW

**Comment by itamar244** [comment]
Adding only changes for CardContent to check if they are good.
Is it good?

**Comment by itamar244** [comment]
Why the test fail?

**Comment by itamar244** [comment]
finished

**Comment by itamar244** [comment]
ready for a merge

File: test/specs/views/Card/CardContent-test.js [hunk 1]
```diff
@@ -24,5 +24,6 @@ describe('CardContent', () => {
     mapValueToProps: val => ({ content: val }),
   })
 
+  common.implementsTextAlignProp(CardContent, ['left', 'center', 'right'])
```
Comment (layershifter) [inline test/specs/views/Card/CardContent-test.js]: Lets use `_.without(SUI.TEXT_ALIGNMENTS, 'justified')` in tests.

File: src/views/Card/CardGroup.js [hunk 2]
```diff
@@ -26,11 +27,13 @@ function CardGroup(props) {
     items,
     itemsPerRow,
     stackable,
+    textAlign,
   } = props
 
   const classes = cx('ui',
```
Comment (layershifter) [inline src/views/Card/CardGroup.js]: Lets add newline there:

```diff
-const classes = cx('ui',
+const classes = cx(
+  'ui',

File: src/views/Card/CardMeta.js [hunk 3]
```diff
@@ -8,14 +9,16 @@ import {
...
 } from '../../lib'
 
 /**
  * A card can contain content metadata.
  */
 function CardMeta(props) {
-  const { children, className, content } = props
-  const classes = cx(className, 'meta')
+  const { children, className, content, textAlign } = props
+  const classes = cx(className, useTextAlignProp(textAlign), 'meta')
```
Comment (layershifter) [inline src/views/Card/CardMeta.js]: Lets make this multiline:

```jsx
const classes = cx(
  className, 
  useTextAlignProp(textAlign),
   'meta',
)
```

**Review by layershifter — CHANGES_REQUESTED** [review]
@itamar244 Thanks for PR :+1: I've add some comments

File: test/specs/views/Card/CardContent-test.js (hunk 1 above)
Comment (itamar244) [reply test/specs/views/Card/CardContent-test.js]: Maybe a new constant equals to `['left', 'center', 'right']` is needed?
It is smaller when bundling and faster.

Comment (layershifter) [reply test/specs/views/Card/CardContent-test.js]: I don't think that there is real necessity. This code doesn't work in production builds.

File: src/views/Card/CardContent.js [hunk 4]
```diff
@@ -26,11 +29,13 @@ function CardContent(props) {
...
     meta,
+    textAlign,
   } = props
 
   const classes = cx(
     className,
     useKeyOnly(extra, 'extra'),
+    useTextAlignProp(textAlign),
     'content',
   )
```
Comment (layershifter) [inline src/views/Card/CardContent.js]: As we update almost all `Card` components, lets also move `className` to end, it's one of our style rules. I overlooked this on initial review.

```jsx
cx(
  useKeyOnly(extra, 'extra'),
  useTextAlignProp(textAlign),
   'content',
  className,
)	
```

**Review by layershifter — CHANGES_REQUESTED** [review]

File: test/specs/views/Card/CardContent-test.js [hunk 5]
```diff
@@ -1,3 +1,6 @@
+import _ from 'lodash';
```
Comment (layershifter) [inline test/specs/views/Card/CardContent-test.js]: Heads up, tests are failing due to lint:

```
/home/ubuntu/Semantic-UI-React/test/specs/views/Card/CardContent-test.js
  1:23  error  Extra semicolon  semi
```

You can run `npm run lint:fix` to fix most errors and then push.  Note that you can ignore warnings.

File: src/views/Card/CardContent.js (hunk 4 above)
Comment (itamar244) [reply src/views/Card/CardContent.js]: Should i first write `'ui'`, then functions, then other literals and then `className`?

**Review by layershifter — APPROVED** [review]
@itamar244 Thanks, awesome work 👍

**Comment by levithomason** [comment]
You guys are great!  Thanks again @layershifter for always filling in for me ❤️

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.73.0`

**Pull request merged**